docs/howto/auth/nativeauth.rst

```diff
@@ -0,0 +1,38 @@
+.. _howto/auth/nativeauth:
+
+==================================================
+Users can signup and create username and password
```

Comment:
What do you think of making the title be 'let users sign up with a username and password'? This way, it completes the sentence 'how do I <title>', which is how other authenticator docs are titled.

Reply:
Just changed it :)